acceptance_tests/features/steps/view_collection_exercise_details.py

```diff
@@ -16,6 +21,17 @@ def internal_user_views_2017_bres_collection_exercise(_):
 @then('the user is able to view the survey details and period for that survey')
 def internal_user_can_view_bres_2017_collection_exercise_details(_):
     ce_details = collection_exercise_details.get_collection_exercise_details()
-    assert ce_details['survey_info'] == "139 - Quarterly Business Survey (QBS)"
+    assert ce_details['survey_info'] == "139 QBS"
+    assert ce_details['survey_title'] == "Quarterly Business Survey"
     assert ce_details['period'] == "1803"
     assert ce_details['user_description'] == "9 March 2018"
+
+
+@then('the user is able to view the event dates for that collection exercise')
+def internal_user_can_view_bres_2017_collection_exercise_events(_):
+    ce_events = collection_exercise_details.get_collection_exercise_events()
+    assert ce_events['mps'] == "Mon 11 Sep 2017 23:00 GMT"
```

Comment:
I guess the data for this will be seeded from here: https://github.com/ONSdigital/rm-tools/blob/3dd720da3f4a21d339f7e9e5284f3dcfb04c4aa0/collex-loader/data/139_2018_schedule.csv#L2 
Would this need to be updated before merging? Not certain what's happening with the data sorry.

Reply:
This is a good point, I'm not sure what the agreement is on where we get our test data from/how we can be consistent using it